theme/lumo/vaadin-confirm-dialog-styles.html

```diff
@@ -92,6 +95,22 @@
           margin-bottom: var(--lumo-space-s);
         }
       }
+
+      /* RTL specific styles */
+
+      [dir="rtl"] > .cancel-button {
+        flex-grow: 1;
```

Comment:
nit: this is already being set at the `.cancel-button` selector at the line `57`.

Reply:
Good catch, remove extra